doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-list-map-add-remove-item-proc.adoc

```diff
@@ -1,19 +1,55 @@
+The test scenarios designer supports list and map collections for both DMN-based as well as rules-based test scenarios. You can create and define a collection like a list or a map ​as the value of a particular cell in both *GIVEN* and *EXPECT* columns.
 
 [NOTE]
 ====
 For map entries, an entry key must be a `String` data type.
 ====
 
+To pass the parameter in the *EXPECT* column of Rule-based collection editor use the `actualValue` keyword whereas use the `?` keyword in DMN-based test scenario.
 
 .Procedure
 . Set the column type first (use a field whose type is a list or a map).
 . Double click a cell in the column to input a value.
-. In the collection editor popup, click *Add new item*.
-. Enter the required value and click the check icon image:dmn/dmn-datatype-constraints-tickmark.png[] to save each collection item that you add.
-. Click *Save*.
+. To create the list values for the data objects in the collection editor popup:
+.. Select *Create List*.
+.. Click *Add new item*.
+.. Enter the required value and click the check icon image:dmn/dmn-datatype-constraints-tickmark.png[] to save each collection item that you add.
+.. Click *Save*.
+.. To edit an item from the collection, click the pencil icon in the collection popup editor.
+.. Click *Save changes*.
+.. To delete an item from the collection, click the bin icon in the collection popup editor.
+
+. To define the list values for the data objects in the collection editor popup:
+.. Select *Define List*.
+.. Use the MVEL or FEEL expression to define a list values in the text field.
++
+Rule-based test scenario uses MVEL expression language and DMN-based test scenario uses FEEL expression language.
+.. Click *Save*.
+
+. To create the map values for the data objects in the collection editor popup:
+.. Select *Create Map*.
+.. Click *Add new item*.
+.. Enter the required value and click the check icon image:dmn/dmn-datatype-constraints-tickmark.png[] to save each collection item that you add.
+.. Click *Save*.
+.. To edit an item from the collection, click the pencil icon in the collection popup editor.
+.. Click *Save changes*.
+.. To delete an item from the collection, click the bin icon in the collection popup editor.
+
+. To define the map values for the data objects in the collection editor popup:
+.. Select *Define Map*.
+.. Use the MVEL or FEEL expression to define a map values in the text field.
++
+Rule-based test scenario uses MVEL expression language and DMN-based test scenario uses FEEL expression language.
+.. Click *Save*.
 +
-To delete an item from the collection, click the bin icon in the collection popup editor. Click *Remove* to delete the collection itself.
+[NOTE]
+====
+You can not use collection editor to define the map values for DMN-based test scenario. To define the map values, do the following:
```

Comment:
I have not seen a procedure in a NOTE, it would be good to make it a single sentence instead of NOTE.

WDYT?

Reply:
How about:
You can add a fact and use the expression with the prefix # to define the map values for DMN-based test scenario, instead of using the collection editor.

Reply:
This makes sense now. 
Even I was not sure about this.